data/json/snippets/graffiti.json

```diff
@@ -107,7 +107,7 @@
       { "id": "shelter_graffiti_29", "text": "NO ONE LEFT HERE MOVE ON" },
       { "id": "shelter_graffiti_30", "text": "deth trap" },
       { "id": "shelter_graffiti_31", "text": "no ones coming" },
-      { "id": "shelter_graffiti_32", "text": "GOVERMENT DID THIS TO US" },
```

Comment:
This typo might actually be intended

Reply:
Right, I always get carried away with the typo hunting season and don't triple-check all my changes x)
good thing reverting is easy